C_CPP/Strings and their functions/reverse_string.md

```diff
@@ -0,0 +1,28 @@
+<h1 text align="center"> Reversing words in a String</h1>
+<p>This a answer for leetcode  <a href"https://leetcode.com/problems/reverse-words-in-a-string/">question</a> https://leetcode.com/problems/reverse-words-in-a-string/ <br>
```

Comment:
Embed the link in the name of the problem question and also write the whole problem here.